recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
   run:
     - python
     - cloudpickle >=1.6.0,<=2.0.0
-    - snowflake-connector-python
+    - snowflake-connector-python>=2.7.4
```

Comment:
We removed the version constraint, with Snowflake's agreement, because we had not yet updated snowflake-python-connector. We now have a valid version of this package and can restore the constraint